docs/kyma-environment-broker/08-02-deprovisioning-kyma-environment.md

```diff
@@ -34,3 +34,15 @@ A successful call returns the operation ID:
    ```
 
 4. Check the operation status as described [here](08-03-operation-status.md).
+
+## Subaccount Cleanup Job
+
+Standard workflow for [BTP Operator](https://github.com/SAP/sap-btp-service-operator) resources is to **keep** them untouched by KEB because users may intend to
+keep the external services provisioned through the BTP Operator still operational. In this case, when calling deprovisioning in the BTP Cockpit, users are informed
+there are still instances provisioned by BTP Operator and it's expected by the user to handle the cleanup.
```

Comment:
```suggestion
The standard workflow for [BTP Operator](https://github.com/SAP/sap-btp-service-operator) resources is to keep them untouched by KEB because users may intend to
keep the external services provisioned through the BTP Operator still operational. In this case, when calling deprovisioning in the BTP Cockpit, users are informed
there are still instances provisioned by BTP Operator, and the user is expected to handle the cleanup.
```